subsys/bluetooth/mesh/scheduler_internal.h

```diff
@@ -18,6 +18,13 @@
 extern "C" {
 #endif
 
+#define MAX_YEAR BIT_MASK(7)
```

Comment:
Isn't this wrong? I read in the spec that the max valid year is 0x64? Same for hour (should be 0x19). You also have to check `entry->action` (which needs special handling because the valid values are not continuous).

Reply:
But is it checked anywhere on the client? It has to be checked in both ends. The stack should prohibit users sending prohibited messages, but also assume that rogue nodes might exist so the server also has to check.

Reply:
> It is not. Should I add this to the fix-commit? But then, this whole sanitizing based on package-fit is no longer relevant

I guess packing function is not good place for scheduler register checking since it doesn't have sense for server side. Server has been already checked it when action has been set.